lib/iris/tests/test_coding_standards.py

```diff
@@ -71,8 +71,8 @@ def test_python_versions():
     This test is designed to fail whenever Iris' supported Python versions are
     updated, insisting that versions are updated EVERYWHERE in-sync.
     """
-    latest_supported = "3.10"
-    all_supported = ["3.8", "3.9", latest_supported]
+    latest_supported = "310"
+    all_supported = ["38", "39", latest_supported]
```

Comment:
We now have more usages of non-dotted versions (the requirements files, tox likes no dot) than dotted versions so it's more easy to have non-dotted here